client/src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,118 @@
+import { Box, makeStyles, Typography } from '@material-ui/core';
+import clsx from 'clsx';
+import React, { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import { Authentication, Button, InputField } from '../../components';
+import { theme } from '../../themes/theme';
+import { useValue, validateEmail, validateString } from '../../utils';
+
+const useStyles = makeStyles((theme) => ({
+  heading: {
+    fontWeight: 'bold',
+  },
+  inputFieldFirst: {
+    marginTop: theme.spacing(10),
+  },
+  inputField: {
+    marginTop: theme.spacing(4),
+  },
+  buttonContainer: {
+    marginTop: theme.spacing(8),
+  },
+  button: {
+    '&:hover': {
+      backgroundColor: theme.palette.secondary.main,
+    },
+  },
+  forgotPassword: {
+    textDecoration: 'none',
+    fontWeight: 'bold',
+    color: theme.palette.secondary.main,
+  },
+  passwordField: {
+    marginBottom: theme.spacing(2.5),
+  },
+}));
+
+const Login = () => {
+  const classes = useStyles();
+  const [email, handleEmail, resetEmail] = useValue('');
+  const [password, handlePassword, resetPassword] = useValue('');
+  const [error, setError] = useState({ type: '', description: '' });
+  const history = useHistory();
+
+  const handleValidation = (event, handler) => {
+    setError({ type: '', description: '' });
+    handler(event);
+  };
+
+  const onSubmit = async (event) => {
+    try {
+      event.preventDefault();
+      if (!validateEmail(email)) {
+        setError({
+          type: 'email',
+          description: email.length === 0 ? 'Email required' : 'Enter valid email',
```

Comment:
Both here and on signup, add the api calls based on the convention you and Allen decide on

Reply:
done